internal/testutils/reference_addon.go

```diff
@@ -31,8 +31,8 @@ type singleton struct {
 }
 
 var (
-	ReferenceAddonImageSetDir   = path.Join(AddonsImagesetDir, "reference-addon")
-	ReferenceAddonIndexImageDir = path.Join(AddonsIndexImageDir, "reference-addon")
+	ReferenceAddonImageSetDir   = path.Join(AddonsImagesetDir(), "reference-addon")
+	ReferenceAddonIndexImageDir = path.Join(AddonsIndexImageDir(), "reference-addon")
```

Comment:
Think this is not used anywhere. I added receiver on singleton and forgot to clean.